client-src/elements/chromedash-feature-table.js

```diff
@@ -338,6 +355,45 @@ class ChromedashFeatureTable extends LitElement {
     return nothing;
   }
 
+  shiftStartBy(delta) {
```

Comment:
The function description could be more accurate than `shiftStartBy`. It reserves the space and load the feature for the next/previous page. How about something like "populateCurrentPage"?

Reply:
I see your point.  I changed it to `loadNewPagination()` page.